src/constants.ts

```diff
@@ -7,3 +7,7 @@ export const SGID_SUPPORTED_GRANT_TYPES: ResponseType[] = ['code']
 export const SGID_AUTH_METHOD: ClientAuthMethod = 'client_secret_post'
 
 export const API_VERSION = 2
+
+// TODO: Replace with https://rules.id.gov.sg/api/rule/eval once it's up
+// Was overwritten with local host URL during testing
+export const SGID_RULES_ENGINE_URL = 'https://rules-stg.id.gov.sg/api/rule/eval'
```

Comment:
Should also allow url override in SgidClient constructor

Reply:
This is important for us to test the different env as well as a good practice.

Reply:
Resolved in [6e9e0ca](https://github.com/opengovsg/sgid-client/pull/65/commits/6e9e0ca1445f1e14c59c7364dc0f9e9a382e8623).